cmd/libs/go2idl/parser/parse_test.go

```diff
@@ -119,8 +127,12 @@ type FooObject interface {
 	CommonObject
 }
 
+
+func proto.AFunc( proto.Object proto.Object ) proto.Frobber {}
```

Comment:
I'm going to merge this one since my other generator depends on it. I'd be happy to improve it if you have better ideas over this template issue.